tests/run/case-class-serializable.scala

```diff
@@ -0,0 +1,6 @@
+case class Foo()
+
+object Test {
+  def main(args: Array[String]): Unit =
+    assert(Foo().isInstanceOf[Serializable])
```

Comment:
I would go further and make the test actually serialize and deserialize a case class and verify the instances are ==

Reply:
Done, we can probably reuse parts of that test for lambdas...